metadata.mk

```diff
@@ -3,15 +3,16 @@
 #################################################################################################
 
 # The version of github.com/projectcalico/go-build to use.
-GO_BUILD_VER = v0.65
+GO_BUILD_VER = v0.66
 
 # Version of Kubernetes to use for tests.
-K8S_VERSION     = v1.23.0
-KUBECTL_VERSION = v1.23.0
+K8S_VERSION     = v1.23.3
+# This is used for lachlanevenson/k8s-kubectl and kubectl binary release.
```

Comment:
Any reason this needs to be v1.23.2 instead of v1.23.3?

Reply:
(my eventual plan is to remove the need for a separate var for this - not sure we have any use case for that)

Reply:
https://hub.docker.com/r/lachlanevenson/k8s-kubectl/tags?page=1&name=1.23.3 this image don't have v1.23.3 yet.